learn-css-grid-by-building-a-magazine/index07.css

```diff
@@ -0,0 +1,44 @@
+.magazine {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  /* The grid-template-areas is used to declare the layout depending on grid-area of elements. The first row of the layout has been done below,
+  make the changes to the property to:
+  Add a second row such that the first two columns belong to the element declared with the grid area of social, the next 2 with title and the last 2 with text-4
```

Comment:
spell out the word `2` in this section. It is misleading with the `text-1` bits

Reply:
I've resolved this locally, would be pushed up with next update